scripts/memory-threshold-list.yaml

```
@@ -0,0 +1,36 @@
+- scenarios:
+    - sample.matter.lock.release
+    - sample.matter.light_bulb.release
+  platforms:
+    - nrf52840dk_nrf52840
+  ram_size: 778240
+  rom_size: 241664
+
+- scenarios:
+    - sample.matter.lock.debug
+    - sample.matter.light_bulb.debug
+  platforms:
+    - nrf52840dk_nrf52840
+  ram_size: 931840
+  rom_size: 241664
+
+- scenarios:
+    - sample.matter.lock.release
+  platforms:
+    - nrf7002dk_nrf5340_cpuapp
+  ram_size: 778240
+  rom_size: 503808
+
+- scenarios:
+    - sample.matter.lock.debug
+  platforms:
+    - nrf7002dk_nrf5340_cpuapp
+  ram_size: 942080
+  rom_size: 503808
+
+- scenarios:
+    - applications.nrf_desktop.z.*
+  platforms:
+    - all
+  rom_related_usage: 256
+  ram_related_usage: 1024
```

Hi @katgiadla
Can we slightly update thresholds here?:
```suggestion
  rom_related_usage: 512
  ram_related_usage: 256
```

---

btw. Can we also monitor sizes of child images (bootloaders or netcore image)?

---

> Hi @katgiadla Can we slightly update thresholds here?:

yes, applied

---

The monitoring is based on the value in twister reports. This value is calculated by a script which parses zephyr.elf file. It means it has knowledge only about the main app. Let's see how useful it will be in action. If people enjoy the feature and keep using it we could think about something more sophisticated which would require some extra scripting and changes to the workflows etc.